packages/map-template/src/components/LegendDialog/LegendDialog.scss

```diff
@@ -0,0 +1,76 @@
+.background {
+    z-index: 10000;
```

Comment:
Can we use one of our existing variables for z-index?

Reply:
✅